.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release to GitHub Container Registry
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Build Docker image
+      run: docker build -t model-serve .
+
+    - name: Log into GitHub Container Registry
+      run: echo "${{ secrets.TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+
+    - name: Tag Docker image
+      run: docker tag model-serve ghcr.io/remla2024-team14/model-serving:latest
+
+    - name: Push Docker image to GitHub Container Registry
+      run: docker push ghcr.io/remla2024-team14/model-serving:latest
+
+    - name: Trigger release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.TOKEN }}
+      with:
+        tag_name: v1.0.0
```

Comment:
Instead of hardcoding this, it may be better to use ` tag_name: ${{ GITHUB_REF }}` for example, where GITHUB_REF is like `refs/tags/v2.3.5` (See https://github.com/actions/create-release)